### PR TITLE
Skip failing test; update maintainers

### PR DESCRIPTION
**Pull request by mfansler**
One of the tests fails due to `tzlocal>=5` - this proposes we skip the test since the failure is due to assumption in the test not the library. This should help unblock the backlog of migrations.

Also, I propose we give the whole R team permissions here. Half the team already is here and this gets R migrations, so I think that simplifies the maintainer scope. However, I'm also happy just to add myself, since I'd like to work the backlog here.

(some minor recipe tidying)

## Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by mfansler** [comment]
@conda-forge-admin please rerender

**Comment by mfansler** [comment]
Ready for review!

@bgruening @ocefpaf does the expansion to R team as maintainers sound good? Otherwise, I'll just add myself.

**Comment by ocefpaf** [comment]
> @bgruening @ocefpaf does the expansion to R team as maintainers sound good? Otherwise, I'll just add myself.

Either is fine IMO.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -76,9 +75,6 @@ about:
 
 extra:
   recipe-maintainers:
-    - bgruening
+    - conda-forge/r
     - ceholden
-    - johanneskoester
-    - mingwandroid
     - ocefpaf
-    - dbast
```
[the remark this answers is not in the document]
Comment (mfansler) [reply recipe/meta.yaml:84]: Ray (@mingwandroid) is no longer around. The other three are in [the R maintainers set](https://github.com/conda-forge/r-feedstock#feedstock-maintainers).

**Pull request merged**